### PR TITLE
Add Symfony 3 & Symfony 4 support

### PR DESCRIPTION
**Pull request by FabienPapet**

### CODE REVIEW

**Comment by FabienPapet** [comment]
This drops the Kitpages/UtilsBundle dependency by adding missing files in the FlysystemBundle

**Comment by FabienPapet** [comment]
ping @philippe-levan

File: composer.json [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
+            "name": "Sebastien Lefebvre"
+        },
+        {
+            "name": "Philippe Le Van",
+            "homepage": "https://twitter.com/#!/plv"
+        }
+    ],
+    "require": {
+        "php": ">=5.6|>=7.0",
+        "symfony/framework-bundle": "~2.8|~3.0|~4.0"
```
Comment (philippe-levan) [inline composer.json]: Is it mandatory to limit to 2.8. Does it work with 2.3 ?

**Review by philippe-levan — COMMENTED** [review]
It is a good idea to remove the dependency. Just check the minimal symfony version.

**Pull request merged**